_posts/0000-01-02-DKP23.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Design can be art. Design can be aesthetics. Design is so simple, that's why it is so complicated.